packages/launch/src/focused-launch/domain-details/index.tsx

```diff
@@ -62,8 +62,8 @@ const DomainDetails: React.FunctionComponent = () => {
 					initialDomainSearch={ domainSearch }
 					onSetDomainSearch={ setDomainSearch }
 					onDomainSearchBlur={ trackDomainSearchInteraction }
-					currentDomain={ selectedDomain?.domain_name || currentDomainName }
-					existingSubdomain={ currentDomainName }
+					currentDomain={ selectedDomain || mockDomainSuggestion( currentDomainName || '' ) }
```

Comment:
When `currentDomainName` is `undefined, the `mockDomainSuggestion()` function should return `undefined` anyway — if so, the ` || '' ` part may be omitted?

Reply:
@ciampo that's right good spot. I will make this change in the morning I thought I had changed all of these.